docker/sanitizers/build_sanitizer.sh

```diff
@@ -92,6 +93,17 @@ DIR_BUILD_EXT="${DIR_EXTERNAL}/cmake.bld"
 DIR_SRC_BMQ="${DIR_ROOT}"
 DIR_BUILD_BMQ="${DIR_SRC_BMQ}/cmake.bld/Linux"
 
+# Create Python venv
+python3 -m venv venv
+if [ -f "venv/bin/activate" ]; then
+    # shellcheck disable=SC1091
+    source venv/bin/activate
+    pip install -r "${DIR_SRC_BMQ}/src/python/requirements.txt"
+else
+  echo "Virtual environment not found."
+  exit 1
```

Comment:
```suggestion
    echo "Virtual environment not found."
    exit 1
```
Small indentation issue